docs/configuration.md

```diff
@@ -113,7 +113,7 @@ Policy scripts are executed by the [CCF JavaScript runtime](https://github.com/m
 Example `set_scitt_configuration` snippet:
 ```json
 "policy": {
-  "policyScript": "export function apply(profile, phdr) {\n  if (profile === \"X509\") { return true; }\n  return \"Only X509 claim profile is allowed\";\n}"
+  "policyScript": "export function apply (phdr) { return true; }"
```

Comment:
Please remove the `Trust stores` section below as users would no longer need to configure x509 roots.